docs/xxidletuningminidlewaittime.md

```diff
@@ -36,7 +36,7 @@ When the OpenJ9 VM is idle, this option controls the minimum length of time that
 
 | Setting     |  Value           | Default  |
 |-------------|------------------|----------|
-|`<secs>`     | *[0 or greater]* | 0        |
+|`<secs>`     | *[0 or greater]* | 180      |
```

Comment:
I think we need to mention two default values, one for non-container case (default is 0) and another for container case (default is 180).